recipes/r-intlim/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set version = 'v.1.1.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-intlim
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - https://github.com/Mathelab/IntLIM/archive/{{ version }}.tar.gz
+  sha256: b0ea63ec73a045037e1bfec035eda1f085f617fdfd339c69e9b4522e841ffe7d 
+
+build:
+  merge_build_host: True  # [win]
+
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
```

Comment:
if this package needs to be compiled, can you please add compilers as well into the build section.

Reply:
I think the compiler `{{ compiler('c') }}` should be explicitly added, right @bgruening?

Reply:
Sorry to ask these maybe simple question: I've checked other comparable recipes and it seems that we could live without having any build step for Windows. The authors of the software indicate that there is no compilation necessary. I can confirm this on my Windows machine when installing it via devtools.
So, my idea was to completely remove the build step.